### PR TITLE
First Draft of the Search Screen and also logo

### PR DESCRIPTION
**Pull request by sahejpg**
Made a logo and also made UI for searching for notes based on the filters we chose. Right now the Main Search screen is on the app screen just so I could see it, but feel free to comment it out whoever is working on the frontend too.

### CODE REVIEW

**Comment by AmanTahiliani** [comment]
I love the Logo! I think the code changes look good to me. 
@afazio1  could comment better on the frontend aspect of it.

**Review by AmanTahiliani — APPROVED** [review]
Looks good to me. Great logo

File: frontend/src/screens/MainScreenWrapper.css
Comment (afazio1) [inline frontend/src/screens/MainScreenWrapper.css:1]: Forgot to push this, but on my branch there is a `src/styles` folder for css files. You can add that folder the your branch and put this file there. Also remember to update the css import!

File: frontend/src/assets/PeerNotes.png
Comment (afazio1) [inline frontend/src/assets/PeerNotes.png:1]: Where is this image used? There seems to be a duplicate one `public/PeerNotes.png`

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,20 @@
 import './App.css'
+import MainSearch from './components/MainSearch';
 
 function App() {
 
   return (
     <>
-      <h1>PeerNotes</h1>
+      <div className="App" style={{ display: 'flex', flexDirection: 'column', justifyContent: 'center', alignItems: 'center', height: '100vh' }}>
```
Comment (afazio1) [inline frontend/src/App.tsx:8]: We could add something like `gap: 1` here for a little bit of spacing between image and MainSearch. Let me know if you think that would look good 👍

File: frontend/src/screens/MainScreenWrapper.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  label {
+    margin-bottom: 5px;
+    font-weight: bold;
+  }
+  
+  select {
+    padding: 8px;
+    border: 1px solid #2E8BC0;
+    border-radius: 4px;
+    width: 100%; /* fill the container width */
```
Comment (afazio1) [inline frontend/src/screens/MainScreenWrapper.css:35]: We could make all selects have a full width if we add `.filter-item { width: 100%; }`

**Review by afazio1 — CHANGES_REQUESTED** [review]
Looks awesome! Left comments about a few minor organization things and some thoughts I had. Feel free to let me know what you think!

File: frontend/src/assets/PeerNotes.png
Comment (sahejpg) [reply frontend/src/assets/PeerNotes.png:1]: yeah there's a duplicate, will get rid of that

**Pull request merged**